content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -5461,10 +5471,18 @@ $connect(): Promise<void>
 
 ### <inlinecode>$on()</inlinecode>
 
+<Admonition type="warning">
+  `$on` is not available in extended clients. Please either migrate to client extensions or use the `$on` method prior to extending your client.
```

Comment:
This needs a link to Client extensions docs that mention that as well. "extended client" does not mean anything to users here.